easybuild/easyconfigs/i/InChI/InChI-1.06.patch

```diff
@@ -0,0 +1,54 @@
+diff -up INCHI-1-SRC/INCHI_API/demos/inchi_main/gcc/makefile.r INCHI-1-SRC/INCHI_API/demos/inchi_main/gcc/makefile
```

Comment:
Please add a short description on top of this patch file to describe what it does, and add a reference to the source or mention the author (and keep in mind that this implies updating the SHA256 checksum for the patch file)

Reply:
The reference is at the top of the file line 7:
\# Notes::       based on Fedora RPM spec file https://src.fedoraproject.org/rpms/inchi/blob/rawhide/f/inchi.spec

I will add a link for the patch there.


Then, how to resubmit the changes in the PR?